GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1850,7 +1840,7 @@ private void SetGitModule(object sender, GitModuleEventArgs e)
                     // If we're applying custom branch or revision filters - reset them
                     RevisionGrid.ResetAllFilters();
                     ToolStripFilters.ClearQuickFilters();
-                    AppSettings.BranchFilterEnabled = AppSettings.BranchFilterEnabled && AppSettings.ShowCurrentBranchOnly;
+                    AppSettings.BranchFilterEnabled = AppSettings.BranchFilterEnabled;
```

Comment:
This does nothing any longer. Or have I missed something?
Why have I approved this?
I am removing it with the upcoming View Settings.

Reply:
I guess we both missed it :)
You're right, it's not longer doing anything.